tests/foreman/ui/test_subscription.py

```diff
@@ -548,3 +548,58 @@ def test_positive_candlepin_events_processed_by_STOMP(session, rhel7_contenthost
         response = entities.Ping().search_json()['services']['candlepin_events']
         assert response['status'] == 'ok'
         assert '0 Failed' in response['message']
+
+
+@pytest.mark.tier2
+def test_positive_manifest_import_delete(session, target_sat):
```

Comment:
This seems like a test coverage that could be handled in the end-to-end test, which effectively already goes through steps 1-4 in this test.  You could simply add some steps to that test function and add the BZ and customerscenario context.

If you disagree and would like to keep this test coverage unique, steps 1-4 are setup for this test function, and should be done in fixtures. The actual test scenario is re-importing, because import+delete are covered elsewhere.